### PR TITLE
added RHash

### PR DESCRIPTION
**Pull request by fredrike**
_Motivation:_ Added RHash.
_Linked issues:_ #2823 

### Checklist
- [x] Build rule `all-supported` completed successfully
- [x] New installation of package completed successfully

### CODE REVIEW

**Comment by cytec** [comment]
uhm... from just a quick look: that doesn't seem to work. you disabled configure_target and just copy a binary file in install? Where does that binary came from? what you are downloading seems to be the source...

File: cross/rhash/Makefile [hunk 1]
```diff
@@ -0,0 +1,23 @@
+PKG_NAME = RHash
+PKG_VERS = 1.3.5
+PKG_EXT = tar.gz
+PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = https://github.com/rhash/RHash/archive
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
```
Comment (cytec) [inline cross/rhash/Makefile]: please use `PKG_DIST_FILE` for a more clear filename than `v1.3.5.tar.gz` something like:
`PKG_DIST_FILE=$(PKG_NAME)-v$(PKG_VERS).$(PKG_EXT)`

see https://github.com/SynoCommunity/spksrc/blob/master/cross/rtmpdump/Makefile#L5 for an example

File: cross/rhash/digests [hunk 2]
```diff
@@ -0,0 +1,3 @@
+v1.3.5.tar.gz SHA1 990c9a0135afe48dee4ce1dbf83578e44735f302
```
Comment (cytec) [inline cross/rhash/digests]: Rerun  `make clean && make digests` after you changed `PKG_DIST_FILE`

File: spk/rhash/src/installer.sh [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+}
+
+postinst ()
+{
+    # Link
+    ln -s ${SYNOPKG_PKGDEST} ${INSTALL_DIR}
+    
+    # Put nano in the PATH
+    mkdir -p /usr/local/bin
+    ln -s ${INSTALL_DIR}/bini/rhash /usr/local/bin/rhash
```
Comment (cytec) [inline spk/rhash/src/installer.sh]: seems like you've got an typo in here shouldn't it be `bin` instead of `bini`

Comment (fredrike) [reply spk/rhash/src/installer.sh]: Sorry, on and airport with shady ssh connection to my buildsystem so managed to sneak in some typos.

**Comment by fredrike** [comment]
@cytec, I've updated accordingly.

**Comment by fredrike** [comment]
@cytec: regarding your old comment. `spksrc` seems to do a lot of magic so although we are not running `./configure` we do indeed run `cd $(WORK_DIR) && make` and that is all that is needed for *RHash* 

<details>
 <summary>Here is the output for  'make arch-x64-5.2'</summary>

```
fer@a8b8e2d5a7bc ~/spksrc/spk/rhash$ make arch-x64-5.2
===>  Building package for arch x64-5.2
make[1]: Entering directory `/home/fer/spksrc/spk/rhash'
===>  Set up toolchain 
make[2]: Nothing to be done for `default'.
make[1]: Leaving directory `/home/fer/spksrc/spk/rhash'
make[1]: Entering directory `/home/fer/spksrc/spk/rhash'
===>  Processing dependencies of rhash
make[2]: Entering directory `/home/fer/spksrc/cross/rhash'
===>  Downloading files for RHash
===>    File RHash-v1.3.5.tar.gz already downloaded
===>  Verifying files for RHash
===>    Checking sha1sum of file RHash-v1.3.5.tar.gz
===>    Checking sha256sum of file RHash-v1.3.5.tar.gz
===>    Checking md5sum of file RHash-v1.3.5.tar.gz
/home/fer/spksrc/cross/rhash/../../distrib/RHash-v1.3.5.tar.gz
===>  Processing dependencies of RHash
===>  Extracting for RHash
tar -xzpf /home/fer/spksrc/cross/rhash/../../distrib/RHash-v1.3.5.tar.gz -C /home/fer/spksrc/spk/rhash/work-x64-5.2 
===>  Patching for RHash
===>  Configuring for RHash
===>  Compiling for RHash
make[3]: Entering directory `/home/fer/spksrc/spk/rhash/work-x64-5.2/RHash-1.3.5'
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c -pipe -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections  -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations calc_sums.c -o calc_sums.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c -pipe -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections  -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations hash_print.c -o hash_print.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c -pipe -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections  -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations common_func.c -o common_func.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c -pipe -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections  -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations hash_update.c -o hash_update.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c -pipe -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections  -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations file_mask.c -o file_mask.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c -pipe -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections  -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations file_set.c -o file_set.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c -pipe -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections  -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations find_file.c -o find_file.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c -pipe -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections  -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations hash_check.c -o hash_check.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c -pipe -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections  -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations output.c -o output.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c -pipe -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections  -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations parse_cmdline.c -o parse_cmdline.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c -pipe -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections  -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations rhash_main.c -o rhash_main.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c -pipe -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections  -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations win_utils.c -o win_utils.o
make -C librhash lib-static
make[4]: Entering directory `/home/fer/spksrc/spk/rhash/work-x64-5.2/RHash-1.3.5/librhash'
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations algorithms.c -o algorithms.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations byte_order.c -o byte_order.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations plug_openssl.c -o plug_openssl.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations rhash.c -o rhash.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations rhash_timing.c -o rhash_timing.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations rhash_torrent.c -o rhash_torrent.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations aich.c -o aich.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations crc32.c -o crc32.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations ed2k.c -o ed2k.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations edonr.c -o edonr.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations hex.c -o hex.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations md4.c -o md4.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations md5.c -o md5.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations sha1.c -o sha1.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations sha256.c -o sha256.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations sha512.c -o sha512.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations sha3.c -o sha3.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations ripemd-160.c -o ripemd-160.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations gost.c -o gost.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations has160.c -o has160.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations snefru.c -o snefru.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations tiger.c -o tiger.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations tiger_sbox.c -o tiger_sbox.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations tth.c -o tth.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations torrent.c -o torrent.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations whirlpool.c -o whirlpool.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc -c  -pipe -DIN_RHASH  -O2 -DNDEBUG -fomit-frame-pointer -ffunction-sections -fdata-sections -Wall -W -Wstrict-prototypes -Wnested-externs -Winline -Wpointer-arith -Wbad-function-cast -Wmissing-prototypes -Wmissing-declarations whirlpool_sbox.c -o whirlpool_sbox.o
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-ar cqs librhash.a algorithms.o byte_order.o plug_openssl.o rhash.o rhash_timing.o rhash_torrent.o aich.o crc32.o ed2k.o edonr.o hex.o md4.o md5.o sha1.o sha256.o sha512.o sha3.o ripemd-160.o gost.o has160.o snefru.o tiger.o tiger_sbox.o tth.o torrent.o whirlpool.o whirlpool_sbox.o
make[4]: Leaving directory `/home/fer/spksrc/spk/rhash/work-x64-5.2/RHash-1.3.5/librhash'
/home/fer/spksrc/toolchains/syno-x64-5.2/work/x86_64-pc-linux-gnu/bin/x86_64-pc-linux-gnu-gcc calc_sums.o hash_print.o common_func.o hash_update.o file_mask.o file_set.o find_file.o hash_check.o output.o parse_cmdline.o rhash_main.o win_utils.o -o rhash -Llibrhash -lrhash  
make[3]: Leaving directory `/home/fer/spksrc/spk/rhash/work-x64-5.2/RHash-1.3.5'
===>  Installing for RHash
find /home/fer/spksrc/spk/rhash/work-x64-5.2/install//usr/local/rhash/ \! -type d -printf '%P\n' | sort > /home/fer/spksrc/spk/rhash/work-x64-5.2/RHash.plist.tmp
find /home/fer/spksrc/spk/rhash/work-x64-5.2/install//usr/local/rhash/ \! -type d -printf '%P\n' | sort | \
          diff /home/fer/spksrc/spk/rhash/work-x64-5.2/RHash.plist.tmp -  | grep '>' | cut -d' ' -f2- > /home/fer/spksrc/spk/rhash/work-x64-5.2/RHash.plist
make[2]: Leaving directory `/home/fer/spksrc/cross/rhash'
===>  Processing wheels of rhash
cat: /home/fer/spksrc/spk/rhash/work-x64-5.2/python-cc.mk: No such file or directory
===>  Creating target installation dir of rhash
(cd /home/fer/spksrc/spk/rhash/work-x64-5.2/install//usr/local/rhash && tar cpf - `cat /home/fer/spksrc/spk/rhash/work-x64-5.2/PLIST | cut -d':' -f2`) | \
          tar xpf - -C /home/fer/spksrc/spk/rhash/work-x64-5.2/staging
===>  Stripping binaries and libraries of rhash
Stripping bin/rhash... ok
(cd /home/fer/spksrc/spk/rhash/work-x64-5.2/staging && tar cpzf /home/fer/spksrc/spk/rhash/work-x64-5.2/package.tgz --owner=root --group=root *)
===>  Creating INFO file for rhash
===>  Creating checksum for rhash
===>  Creating /home/fer/spksrc/spk/rhash/work-x64-5.2/scripts/preinst
===>  Creating /home/fer/spksrc/spk/rhash/work-x64-5.2/scripts/postinst
===>  Creating /home/fer/spksrc/spk/rhash/work-x64-5.2/scripts/preuninst
===>  Creating /home/fer/spksrc/spk/rhash/work-x64-5.2/scripts/postuninst
===>  Creating /home/fer/spksrc/spk/rhash/work-x64-5.2/scripts/preupgrade
===>  Creating /home/fer/spksrc/spk/rhash/work-x64-5.2/scripts/postupgrade
===>  Creating /home/fer/spksrc/spk/rhash/work-x64-5.2/scripts/installer
(cd /home/fer/spksrc/spk/rhash/work-x64-5.2 && tar cpf /home/fer/spksrc/spk/rhash/../../packages/rhash_x64-5.2_1.3.5-1.spk --group=root --owner=root package.tgz INFO scripts)
make[1]: Leaving directory `/home/fer/spksrc/spk/rhash'
</details>

**Comment by cytec** [comment]
@fredrike thanks ;) WRT my old comment: yeah my mistake that popped in my mind the moment i've hit the comment button

File: spk/rhash/Makefile [hunk 4]
```diff
@@ -0,0 +1,21 @@
+SPK_NAME = rhash
+SPK_VERS = 1.3.5
+SPK_REV = 1
+
+DEPENDS = cross/$(SPK_NAME)
+
+MAINTAINER = Fredrike
+DESCRIPTION = "RHash is a console utility for calculation  and verification of magnet linksand a wide range of hash sums like  CRC32,  MD4,	 MD5,  SHA1, SHA256, SHA512,SHA3,   AICH,  ED2K,  Tiger,  DC++ TTH,  B	itTorrent BTIH,   GOST R 34.11-94,RIPEMD-160, HAS-160, EDON-R, Whirlpo	ol and Snefru."
```
Comment (Safihre) [inline spk/rhash/Makefile]: Maybe double-check this text? I think it should be:
_RHash is a console utility for calculation and verification of magnet links and a wide range of hash sums like CRC32,  MD4, MD5, SHA1, SHA256, SHA512, SHA3, AICH,  ED2K, Tiger, DC++ TTH, BitTorrent BTIH, GOST R 34.11-94, RIPEMD-160, HAS-160, EDON-R, Whirlpool and Snefru._

Comment (fredrike) [reply spk/rhash/Makefile]: Thanks for spotting this.

**Comment by ymartin59** [comment]
@cytec I find it "too small" to create a dedicated package for each little command line utilities. There was an old request for that: #2261
And there are lots of PR pending with such tools: bc, less, ntop, nmon, lsscsi...

So question: isn't time to create this "CLI Tools" package ("GNU" if relevant but maybe not so essentials by the way...) ?

**Comment by fredrike** [comment]
Sorry for spending time on this. Why have the package request #2823 not a status saying that it will not be merged?

**Comment by Safihre** [comment]
That's only @ymartin59's opinion, what does @cytec think? :)

**Comment by cytec** [comment]
@fredrike your time wasn't wasted no worries! as @ymartin59 stated it's a little bit "small" for a own SPK but from my point of view nothing speaks against merging it and start working on those grouped SPK's mentioned in https://github.com/SynoCommunity/spksrc/issues/2261 

we just have to get those group names straight, merge the PR's and build grouped SPK's currently im more or less satisfied with the approach @Dr-Bean stated here https://github.com/SynoCommunity/spksrc/issues/2261#issuecomment-211224194 we just have to find a solid way of grouping the tools together and a naming scheme for the SPK's

**Comment by m4tt075** [comment]
@fredrike Many thanks for your contribution! I'm looking into merging your work as part of a bigger `SynoCLI-file` package. If you are still interested, please feel free to track any progress in issue #3250 and PR #3598. 

For everyone, test-builds of the `SynCLI-file` package for all currently supported DSM5.2, 6.1 and 6.2 platforms are available here:  https://github.com/m4tt075/spksrc/releases/tag/synoclifile-2
Please report any issues in #3250, not here.

**Pull request closed without merging**